src/EnergyPlus/SQLiteProcedures.hh

```diff
@@ -56,10 +56,10 @@ public:
 	// This allows for testing of private methods in SQLite
 	friend class SQLiteFixture;
 
-	void addScheduleData( int const number, std::string const name, std::string const type, double const minValue, double const maxValue );
+	void addScheduleData( int const number, std::string const & name, std::string const & type, double const minValue, double const maxValue );
```

Comment:
@DeadParrot I remember purposefully keeping these string parameters as non-reference because either unit tests or manual testing showed that these strings were deallocated before writing to SQLite database, causing junk values to be written. I believe this showed up in the Linux unit tests but I can't remember. We can wait and see.

Another possibility is that I fixed the failing unit tests another way and this will work, but we still need to double check the output sql from a full run.

Was there a performance difference using pass by reference for these specific instances? Or was this an automated sweep of the code base?